store/src/main/java/com/dropbox/android/external/store4/impl/SourceOfTruth.kt

```diff
@@ -25,7 +25,7 @@ import kotlinx.coroutines.flow.flow
  * [RealStore] implementation to avoid dispatching values to downstream while
  * a write is in progress.
  */
-internal interface SourceOfTruth<Key, Input, Output> {
+interface SourceOfTruth<Key, Input, Output> {
```

Comment:
why does this need to be public? the rx binding stuff shouldn't require access to non-public API from store

Reply:
fixed

Reply:
maybe you didn't push the changes? it still missing the `internal` modifier (so does the other ones above)